requirements.txt

```diff
@@ -1,46 +1,114 @@
+#
+# This file is autogenerated by pip-compile
```

Comment:
FYI @antoviaque to keep you in the loop on what's different between Ocim and Accounting -- I'm using `pip-tools` to keep a full list of the actual requirements used in the project, i.e. below. It's pretty useful, giving us `pip-sync` for example, to keep our virtualenvs clean.

Reply:
Yup, thanks for letting me know - I wish we had used this for Ocim too, it would make it easier to do updates.

Reply:
It's still possible, why not! Just change the name of `requirements.txt` to `requirements.in`, add `pip-tools`, and run `pip-compile`.

Reply:
Yup, that would be a good start, even if that doesn't show the core requirements at least it would start tracking this a bit better. I don't have time to do this now, but maybe this could be part of the scope on the next upgrade of requirements of Ocim? @smarnach btw I think we have a pending Django upgrade, right? Maybe worth adding a ticket to the backlog for it, and include this at the same time?

Reply:
@UmanShahzad @antoviaque I've used pipenv for the Cloudera Zoho reports app, and I highly recommend it over pip-compile; see https://github.com/open-craft/cloudera-zoho-reports#pipenv for more information.  I think using a `Pipfile` instead of `requirements.txt` is also where the community is going, so my prediction is that this is where we will end up in the long run anyway.

And regarding the requirements of Ocim, yes, we have several pending upgrades: Python, Django, lots of other stuff.  I'll create a ticket once we've agreed on a tool to use.

Reply:
@smarnach Cool, I didn't knew about pipenv, it does look really nice. That would work for me, if @UmanShahzad likes it too for the accounting app.

Reply:
I guess since pypa is backing pipfile it will probably become the standard at some point, yes. But let's leave this for another time, as the 'migration' should be pretty straightforward but kind of tedious and I wouldn't have time for it atm.

Reply:
@UmanShahzad OK - then could you put this in a ticket to not forget about it? It could be a ticket for the next upgrade, like for Ocim. It doesn't require one now, but it will eventually, and this way we'll remember to do this at the same time.

Reply:
@antoviaque Put it on my list! Will do by Friday when planning.